### PR TITLE
Default target selector to devices

### PR DESCRIPTION
**Pull request by kflores772**

### CODE REVIEW

**Comment by kflores772** [comment]
#198 

@rcruz r?

**Comment by rcruz** [comment]
r+

File: lib/ui/views/targetSelector.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ TargetSelectorView = Backbone.View.extend({
         var view = this,
             template;
 
+        if (typeof this.device === "undefined") {
+            this.device = true;
```
Comment (rcruz) [inline lib/ui/views/targetSelector.js:32]: Could also be this.device = this.device || true;

Comment (bryanhiggins) [inline lib/ui/views/targetSelector.js:32]: Or just device: true as a property of this object :)

**Pull request closed without merging**